webapp/src/app/routes/yaml/yaml.interface.ts

```diff
@@ -0,0 +1,81 @@
+export interface Yaml {
```

Comment:
This comment holds a couple of similar comments:

I made a diff of the current yaml and the one our tool outputs:
https://www.diffnow.com/files/__edp_output__3efc2fac-23ae-4347-81dc-110623b3aa64.htm

I saw some differences:
1. params is a list:
https://github.com/hasadna/reportit-scripts/blob/master/src/user/script.yaml#L35
It seems that for us, it's a single field. At least in the UI.

2. Can we control the order?
In `case` for example, it's:
```
steps
default
match
```
Can we put `match` first?

3. If `default` is false, can we not output it?
4. If `default` is true, can we not output `match`?
5. Why does `say` output `>-`?

Thanks!!

Reply:
If any of this requires non-trivial changes, you can do it in the next PR.